src/components/ImportStatus.js

```diff
@@ -47,24 +47,24 @@ const ImportStatusItem = ({ job: { targetWorkspace, jobId }, onDone }) => {
     // avro-import statuses: PENDING, RUNNING, SUCCESS, ERROR
     // import service statuses: Pending, Translating, ReadyForUpsert, Upserting, Done, Error
     // TODO: only need to support both sets of statuses during the transition from avro-import to import service.
-    // once import servie is fully adopted, we can/should remove the avro-import status values.
+    // once import service is fully adopted, we can/should remove the avro-import status values.
 
     const successNotify = () => notify('success', 'Data imported successfully.',
       {
         message: `Data import to workspace "${namespace} / ${name}" is complete, please refresh the Data view.
-      Because this data comes from a PFB, you must include the namespace "pfb:" when referring to attribute names, 
+      If your data comes from a PFB, you must include the namespace "pfb:" when referring to attribute names, 
```

Comment:
if we wanted this message to be tailored to TSV vs. PFB, what would be involved? We'd probably have to persist the filetype into the `asyncImportJobStore`, right?

Reply:
Yeah I tried to figure out a way to do that here, but at this stage we don't know what the file type is. If we could preserve file type and return it when we get/list jobs, that would help here.

Reply:
Ah, yeah - especially after reloading the page, you'd need the filetype from the job list. Got it - another ticket to add to the queue!